packages/neuron-ui/src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
It will throw an error ` React is not defined` without `import React`

Reply:
Got it, we're still using React 16 and the update is imported since React 17

Ref: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html